torch_frame/utils/io.py

```diff
@@ -1,10 +1,48 @@
-from dataclasses import fields
+from dataclasses import asdict
 from typing import Any, Dict, Optional, Tuple
 
 import torch
+from torch import Tensor
 
-from torch_frame.data import TensorFrame
+from torch_frame.data import MultiNestedTensor, TensorFrame
 from torch_frame.data.stats import StatType
+from torch_frame.typing import TensorData
+
+
+def serialize_feat_dict(feat_dict: Dict[str, TensorData]) -> Dict[str, Any]:
+    feat_seriealized_dict = {}
+    for stype, feat in feat_dict.items():
+        if stype.use_multi_nested_tensor:
+            assert isinstance(feat, MultiNestedTensor)
+            feat_seriealized = asdict(feat)
+        elif stype.use_dict_multi_nested_tensor:
+            feat_seriealized = {}
+            assert isinstance(feat, dict)
+            for name, f in feat.items():
+                assert isinstance(f, MultiNestedTensor)
```

Comment:
Same as above?

Reply:
For this one, we only have dict for multi-nested tensor, so we can keep it.